### PR TITLE
Add support for Iblinds

### PR DESCRIPTION
**Pull request by mike240se**
Includes fixes recommended by manufacturer here:

https://support.myiblinds.com/community/home-assistant/configuring-home-assistant-and-iblinds/
and 
https://community.home-assistant.io/t/iblinds-disabling-a-z-wave-command/134545/4

which includes setting getsupport to false and setting level max to 255.

Unfortunately it will still show as a dimmer in HA until the ozwcache is edited and specific is set to 6 (cover).  I was told its not possible to change this in the xml, must be fixed in HA code.

EDIT: also, the company name in manufacturer_specific.xml (empty block) is HAB Home LLC but the zwave alliance DB says the company name should be iblinds so i used iblinds.  if you check the zwave conformance certificate, its issued to company iblinds. not HAB Home LLC

### CODE REVIEW

File: config/iblinds/ib2.xml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    <MetaDataItem name="InclusionDescription">Follow the directions for your Z-Wave hub/controller to place it in the add for inclusion mode. 
+Press the inclusion button once and wait for the device to be discovered and joined to the Z-Wave network.  Follow the directions of your Z-Wave controller to uniquely identify your iblinds device.   (Ex: Bedroom iblind)
+Note: If the device is not joined to the network.  Press the CLBR button Wait 1 minutes and repeat steps 1 and 2 again.  If you continue to have trouble joining the iblinds motor to the Z-Wave network follow the Z-Wave network Exclusion process below.  
+</MetaDataItem>
+    <ChangeLog>
+      <Entry author="mike240se" date="10 Jul 2020" revision="1">Initial metadata</Entry>
+    </ChangeLog>
+  </MetaData>
+   <!-- Configuration Parameters -->
+	<CommandClass id="38">
```
Comment (Fishwaldo) [inline config/iblinds/ib2.xml:29]: This Block Should not be required. Why did you include it?

Comment (mike240se) [reply config/iblinds/ib2.xml:29]: Just to change the help text.  the default talks about dimmers (this device should be a cover but due to a bug in the firmware or design decision its a dimmer).

For example by default it says "Increase the brightness of the device", i changed the help text to "Increase the blinds tilt"

Should I remove it?

**Comment by mike240se** [comment]
just following up if i should remove the help text?

**Comment by mike240se** [comment]
closing to evaluate new way of fixing the getsupported=false issue.  also found typos.

**Pull request closed without merging**